MathlibTest/positivity.lean

```diff
@@ -380,6 +381,13 @@ example (f : D → E) (c : ℝ) (hc : 0 < c): 0 ≤ ∫ x, c * ‖f x‖ ∂μ :
 
 end Integral
 
+/-! ## Infinte Sums -/
+
+example (f : ℕ → ℝ) : 0 ≤ ∑' n, (f n)^2 := by positivity
+example [LinearOrderedField α] [TopologicalSpace α] [OrderClosedTopology α] (f : ℚ → α) :
+    0 ≤ ∑' q, (f q)^2 := by positivity
```

Comment:
This line-break is funny. I know why you do it... perhaps break before the `(f : ...)` instead?

And put a newline after this declaration, please.